core/contracts/oracle/implementation/Voting.sol

```diff
@@ -133,6 +132,14 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
 
     event VoteCommitted(address indexed voter, uint indexed roundId, bytes32 indexed identifier, uint time);
 
+    event EncryptedVote(
+        address indexed voter,
+        uint indexed roundId,
+        bytes32 indexed identifier,
+        uint time,
```

Comment:
If we really wanted to be able to filter on identifier-time pairs, we could form some sort of `topicHash` that's a hash of `time` and `identifier`, and make that `bytes32` an indexed parameter here instead of including both `identifier` and `time`. Right now, you could only filter on `identifier` since time isn't indexed.

I actually think the way you have it now is preferable for all of our use cases. I can't imagine when it'd be important for us to be able to query just based on identifier-time pairs, but wanted to mention it as an alternative in case there was some situation you could think of.

Reply:
A topic hash could work but I think, like you said, that we can achieve the same with how @ptare 's event is set up now.

Reply:
If we did that, we should probably be consistent and make that change for all the events (VoteCommitted, VoteRevealed, etc.). It would definitely make some queries more convenient, but there might also be an advantage to having contract events be somewhat readable. Happy to go in either direction.

Reply:
Totally agree. I think the way it currently works is preferable then, since there are no queries that seem impossible or very inconvenient.